fluix/web-apidoc2ts
# fix/PNG to SVG change (site deploy)

#### Short description
Removed default react app comments from index.html and actually committed new site (unlike #54)

#### How to review
Literally only the first commit.

#### How to test it
You can run site locally using `npm start` but you should change homepage to empty string before doing that.
